MANUAL_QA.md

```
@@ -279,7 +287,105 @@ test-team-1/general/testuser > Line 1 Line 2 Line 3
 [INFO] All connections closed.
 ```
 
-### 5.7 パイプ・リダイレクトテスト
+### 5.7 ハイライト機能テスト
+
+#### 5.7.1 キーワードマッチングテスト
+
+**作業内容**: 設定したキーワードを含むメッセージを投稿する
+
+**実行手順**:
+1. 設定ファイルに以下のキーワードが設定されていることを確認：
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)